sklearn/tests/test_metaestimators.py

```diff
@@ -197,6 +198,52 @@ def score(self, X, y, *args, **kwargs):
             )
 
 
+def _get_instance_with_pipeline(meta_estimator, init_params):
+    """Given a single meta-estimator instance, generate an instance with a pipeline"""
+    if {"estimator", "base_estimator", "regressor"} & init_params:
+        if is_regressor(meta_estimator):
+            estimator = make_pipeline(TfidfVectorizer(), Ridge())
+            param_grid = {"ridge__alpha": [0.1, 1.0]}
+        else:
+            estimator = make_pipeline(TfidfVectorizer(), LogisticRegression())
+            param_grid = {"logisticregression__C": [0.1, 1.0]}
+
+        if {"param_grid", "param_distributions"} & init_params:
```

Comment:
```suggestion
        if init_params.intersection({"param_grid", "param_distributions"}):
```

Reply:
But this is the same operator that is run under the hood. I just find it easier to have the English name rather than trying to understand why you are doing a bit-wise operation between parameter names and then find out that they are sets and it make sense :)